### PR TITLE
Introduction to hapi.js framework

### PR DESCRIPTION
**Pull request by kayere**
Added introduction to hapi.js framework article.

### CODE REVIEW

File: articles/introduction-to-hapi.js-framework/index.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+//start the server
+server.start(error => {
+    if (error) throw error
+    else console.log('Server running at PORT 2400');
+})
+```
+You can run your server by running node 'name of your index file' on the terminal.
+
+### Creating routes with hapi.js
+Hapi uses server.route as a method to create routes. server.route methods take an options object. The object has three properties.
```
Comment (hectorkambow) [inline articles/introduction-to-hapi.js-framework/index.md:80]: @kayere if you could please elaborate this couple sentences - they may be good - but as a beginner myself - I found myself kind of asking more questions to myself while reading theses few sentences. If you could break down so any beginner can understand what an option object is

Comment (kayere) [reply articles/introduction-to-hapi.js-framework/index.md]: Hello @hectorkambow , I have elaborated the few sentences as requested. Please review. Thanks.

**Comment by hectorkambow** [comment]
Good afternoon @gwritz  - wanted to ask if you could give this Hapi.js article a tech review before we publish. 
Thank you 🚀

File: articles/introduction-to-hapi.js-framework/index.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+Let's see how we can create a simple server using hapi.
+
+First run
+
+```
+npm init
+```
+To create a node package with the name of your choice.
+
+Then open a new terminal and navigate to your app's folder and run
```
Comment (gwritz) [inline articles/introduction-to-hapi.js-framework/index.md:53]: These commands are out of order. You'd need to open a terminal and create a directory for your app before running npm init and the npm install.

Comment (kayere) [reply articles/introduction-to-hapi.js-framework/index.md:53]: Noted... Let me make the changes right away.

**Pull request merged**